RevenueCatUI/CustomerCenter/Views/AppUpdateWarningView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  AppUpdateWarningView.swift
+//
+//  Created by JayShortway on 16/08/2024.
+
+#if CUSTOMER_CENTER_ENABLED
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct AppUpdateWarningView: View {
+    let onUpdateAppClick: () -> Void
+    let onContinueAnywayClick: () -> Void
+
+    @Environment(\.localization)
+    private var localization: CustomerCenterConfigData.Localization
+    @Environment(\.appearance)
+    private var appearance: CustomerCenterConfigData.Appearance
+    @Environment(\.colorScheme)
+    private var colorScheme
+
+    @ViewBuilder
+    var content: some View {
+        ZStack {
+            List {
+                Section {
+                    CompatibilityContentUnavailableView(
+                        localization.commonLocalizedString(for: .updateWarningTitle),
+                        systemImage: "arrowshape.up.circle.fill",
```

Comment:
This image is iOS 17+ only. You can check in Xcode with this dialog (command-shift-L):

<img width="669" alt="image" src="https://github.com/user-attachments/assets/df8b7d8d-c210-4676-ac14-d65cba654451">

Reply:
Ah TIL, thanks! Updated here: https://github.com/RevenueCat/purchases-ios/pull/4193/commits/5e3bebeae7cf6dc0a77752dd7b8bc8d3f25f4361.